packages/addons/service/proftpd/package.mk

```diff
@@ -33,30 +34,26 @@ PKG_CONFIGURE_OPTS_TARGET="--enable-static \
                            --enable-ipv6 \
                            --enable-nls \
                            --enable-pcre \
-                           --enable-largefile"
-
-pre_build_target() {
-  mkdir -p $PKG_BUILD/.$TARGET_NAME
-  cp -RP $PKG_BUILD/* $PKG_BUILD/.$TARGET_NAME
-}
+                           --enable-largefile \
+                           install_user=$(id -u) \
+                           install_group=$(id -g)"
```

Comment:
Is this going to be a problem with "reproducible builds"? What is this used for - does it affect the final binary?

Reply:
This was required because we now use `make install` and upstream installs file for a specific user (chmod). But that requires root privs, except when the uid/gid matches the current user - which this just does to make `make install` succeed

Reply:
Ok thanks.